### PR TITLE
HARVESTER:   single cluster feature & bug fix

### PR DESCRIPTION
**Pull request by WuJun2016**

### CODE REVIEW

**Review by richard-cox — CHANGES_REQUESTED** [review]
Two super minor comments, otherwise LGTM

File: mixins/harvester-vm/impl.js [hunk 1]
```diff
@@ -102,6 +102,17 @@ export default {
       }
     },
 
+    isEfiEnabled(spec) {
+      const smmEnabled = spec?.template?.spec?.domain?.features?.smm?.enabled;
+      const efiEnabled = spec?.template?.spec?.domain?.firmware?.bootloader?.efi?.secureBoot;
+
+      if (smmEnabled && efiEnabled) {
```
[the remark this answers is not in the document]
Comment (n313893254) [reply mixins/harvester-vm/impl.js:109]: smmEnabled and efiEnabled might be `undefined`. Seems `return !!(smmEnabled && efiEnabled)` will be better.

**Comment by WuJun2016** [comment]
Added a few new commits,  PTAL,  thanks.

File: models/harvester/kubevirt.io.virtualmachine.js [hunk 2]
```diff
@@ -798,7 +800,7 @@ export default class VirtVm extends SteveModel {
   }
 
   get stateDescription() {
-    if (this.isOff) {
+    if (this.isOff || /pod has unbound immediate PersistentVolumeClaims/.test(super.stateDescription)) {
```
[the remark this answers is not in the document]
Comment (WuJun2016) [reply models/harvester/kubevirt.io.virtualmachine.js]: Done

File: mixins/harvester-vm/impl.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (WuJun2016) [reply mixins/harvester-vm/impl.js]: Done

File: assets/translations/zh-hans.yaml [hunk 3]
```diff
@@ -5433,6 +5433,7 @@ harvester:
       tips: 凭证生效需要安装qemu-guest-agent，更新凭证内容后无需重启，添加凭证后需要重启生效，删除凭证后，需要进入虚拟机手动修改密码/删除ssh key
```
[the remark this answers is not in the document]
Comment (WuJun2016) [reply assets/translations/zh-hans.yaml:5434]: Done

**Review by richard-cox — APPROVED** [review]

**Pull request merged**